torch/csrc/jit/passes/remove_inplace_ops.cpp

```diff
@@ -42,9 +48,21 @@ void RemoveInplaceOps(Block* block) {
         newNode->addInput(input);
       }
 
+      size_t additionalInputCount = 0;
+      if (additionalNoneArgumentCount.find(node->kind()) != additionalNoneArgumentCount.end()) {
+        additionalInputCount = additionalNoneArgumentCount.at(node->kind());
+      }
+
+      for (size_t i = 0; i < additionalInputCount; ++i) {
+        auto noneNode = graph->createNone();
+        noneNode->insertBefore(newNode);
+        newNode->addInput(noneNode->output());
+      }
+
       // Create a new output node and replace all uses of self with it
       newNode->output()->copyMetadata(node->output());
       node->replaceAllUsesWith(newNode);
+      node->inputs()[0]->replaceAllUsesAfterNodeWith(newNode, newNode->output());
```

Comment:
Seems we only have one use case for this replaceAllUsesAfterNodeWith, could you inline it here instead of adding it to IR directly at this moment? Later if we have more use cases, we can extract it out.

Reply:
The index put PR also needs this function. Also this function updates `uses_`, which is private under struct Value.